src/test/app/common/court-calculations/courtDetermination.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai'
+
+import { CourtDetermination, DecisionType } from 'common/court-calculations/courtDetermination'
+import { LocalDate } from 'forms/models/localDate'
+
+describe('CourtDetermination', () => {
+
+  context('calculateDecision', () => {
+    it('should return a claimant decision type when claimantPaymentDate is after the defendantPaymentDate', () => {
+
+      let defendantPaymentDate = new LocalDate(2019, 10, 10).toMoment()
+      let claimantPaymentDate = new LocalDate(2019, 10, 11).toMoment()
+      let courtGeneratedPaymentDate = new LocalDate(2019, 10, 12).toMoment()
+
+      expect(CourtDetermination.calculateDecision(defendantPaymentDate, claimantPaymentDate, courtGeneratedPaymentDate)).to.equal(DecisionType.CLAIMANT)
+    })
+
+    it('should return a claimant or defendant decision type when claimantPaymentDate and defendantPaymentDate are the same', () => {
+
+      let defendantPaymentDate = new LocalDate(2019, 12, 31).toMoment()
+      let claimantPaymentDate = new LocalDate(2019, 12, 31).toMoment()
+      let courtGeneratedPaymentDate = new LocalDate(2019, 10, 11).toMoment()
+
+      expect(CourtDetermination.calculateDecision(defendantPaymentDate, claimantPaymentDate, courtGeneratedPaymentDate)).to.equal(DecisionType.CLAIMANT)
+    })
```

Comment:
Since its valid to return either Claimant or Defendant decision type, you could probably try something like this:

`expect(Object.values(DecisionType).includes(CourtDetermination.calculateDecision(defendantPaymentDate, claimantPaymentDate, courtGeneratedPaymentDate)))`